minitftpad.py

```diff
@@ -0,0 +1,204 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2019 for Adafruit Industries LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# We have a lot of attributes for this complex library.
+# pylint: disable=too-many-instance-attributes
+
+"""
+Adafruit Mini Color TFT with Joystick FeatherWing
+`adafruit_featherwing.minitftpad`
+====================================================
+
+CircuitPython driver for `Mini Color TFT with Joystick FeatherWing
+<https://www.adafruit.com/product/3321>`_.
+
+* Author(s): Kattni Rembor, Scott Shawcroft, Radomir Dopieralski, Michael McWethy, Jonah Yolles-Murphy, Melissa LeBlanc-Williams
+"""
+
+from adafruit_seesaw import digitalio as dio
+from adafruit_seesaw import seesaw as ss
+from adafruit_seesaw import pwmout as pwm
+import displayio
+from micropython import const
+import board
+import sys
+# pylint: disable=wrong-import-position
+try:
+    lib_index = sys.path.index("/lib")        # pylint: disable=invalid-name
```

Comment:
Are you doing this for the seesaw library? If so, was there something about the frozen one that wasn't working?

Reply:
Yeah you can make w/e changes you want. I was just kinda making this https://github.com/TG-Techie/TFT-FeatherWing-Lib  into something more displayio.

I'm new to coding so i just pretty much copy paste and then change around other peoples stuff.... Like i have no idea why any underscores are used    =\